**The Academy Awards**, also known as the **Oscars**, are awards given annually by the Academy of Motion Picture Arts and Sciences (AMPAS) in the United States to recognize achievements in cinematic excellence. The awards are considered the most prestigious in the film industry. The Academy's voting membership assesses the achievements, which were first presented in 1929.

The awards recognize excellence in cinematic achievements. To be eligible, a film has to be exhibited in a theater in Los Angeles for at least a week. The exceptions to this are foreign-language movies, which are submitted by their country of origin and need not be featured in the US.

# **Objective:**
## **To understand the trends and patterns of winning categories in Oscar-nominated films over the past 96 years.**.

##1. Importing Libraries & accessing the data

In [1]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt
import seaborn as sns
import plotly.express as px

In [2]:
from google.colab import drive
drive.mount('/content/drive')

Mounted at /content/drive


In [3]:
oscar=pd.read_csv('/content/drive/MyDrive/OSCARS.csv')
oscar.head()

year_film  year_ceremony  ceremony category                 name  \
0       1927           1928         1    ACTOR  Richard Barthelmess   
1       1927           1928         1    ACTOR        Emil Jannings   
2       1927           1928         1  ACTRESS       Louise Dresser   
3       1927           1928         1  ACTRESS         Janet Gaynor   
4       1927           1928         1  ACTRESS       Gloria Swanson   

               film  winner  
0         The Noose   False  
1  The Last Command    True  
2   A Ship Comes In   False  
3        7th Heaven    True  
4    Sadie Thompson   False

In [4]:
oscar.shape

(10889, 7)

In [5]:
oscar.isnull().sum()

year_film          0
year_ceremony      0
ceremony           0
category           0
name               5
film             319
winner             0
dtype: int64

##2. Data Exploration & Analysis

#1. How many unique films, categories, and names are there in the dataset?

In [6]:
#How many unique films?
len(oscar['film'].unique())

5042

In [7]:
#How many unique categories?
len(oscar['category'].unique())

115

In [8]:
#How many unique people?
len(oscar['name'].unique())

7040

### Insights:
1. A total of 5,042 movies were nominated over the period of 96 years.

2. Nominations varied across 115 distinct categories.

3. 7,040 individuals have been nominated, so far

# 2. Out of the distinct films, categories and names nominated, how many won the award?

In [9]:
#Films
win_film=oscar[oscar['winner']==True]['film'].unique()
print("Number of films that won oscar award:", len(win_film))

print(round(len(win_film)/len(oscar['film'].unique()),2)*100,"%", "of nominated films won oscars.")

Number of films that won oscar award: 1329
26.0 % of nominated films won oscars.


In [10]:
#Categories
win_categories=oscar[oscar['winner']==True]['category'].unique()
print("Number of categories that won oscar award:", len(win_categories))

print(round(len(win_categories)/len(oscar['category'].unique()),2)*100,"%", "of nominated categories won oscars.")
print("Thus, each of the categories, have won an award at any certain point of time or the other.")

Number of categories that won oscar award: 115
100.0 % of nominated categories won oscars.
Thus, each of the categories, have won an award at any certain point of time or the other.


In [11]:
#Individuals
win_artist=oscar[oscar['winner']==True]['name'].unique()
print("Number of artists that won oscar award:", len(win_artist))

print(round(len(win_artist)/len(oscar['name'].unique()),2)*100,"%", "of nominated artists won oscars.")

Number of artists that won oscar award: 2082
30.0 % of nominated artists won oscars.


### Insights:

1. Out of 5042 films, 1329 have won an Oscar, depicting a 26.0 % winning proportion.

2. Out of 115 categories nominated, all of them bagged an award, at a certain point in time or at other.

3. 2,082 individuals out of 7,040 (i.e., almost 30%), have won an award.  

#3. What is the distribution of the 'winner' column? How many winners across all categories?


In [12]:
oscar['winner'].value_counts()

False    8424
True     2465
Name: winner, dtype: int64

In [13]:
(2465/10889)*100

22.63752410689687

# Insights:

**Approximately 22.64% of nominations have won the award, this indicates a selective and competitive nature of the accolade**

#4. Trends over time:

##a. How many films released per year?
##b. Are there any trends in the types of categories nominated over the years?
##c. What are the distinctive Nominated Categories & its respective Winning Proportion?

##a. Number of films released per year

In [14]:
films_per_year=oscar['year_film'].value_counts().sort_values('index', ascending=True).reset_index()
films_per_year

<ipython-input-14-69e4a33af2d8>:1: FutureWarning: In a future version of pandas all arguments of Series.sort_values will be keyword-only.
  films_per_year=oscar['year_film'].value_counts().sort_values('index', ascending=True).reset_index()


index  year_film
0    1927         35
1    1928         38
2    1929         42
3    1930         44
4    1931         45
..    ...        ...
91   1945        160
92   1944        169
93   1943        182
94   1941        183
95   1942        186

[96 rows x 2 columns]

In [15]:
px.bar(films_per_year,x='index', y='year_film', color='year_film', title='Films Released Per Year')

##b. Trend in nominated categories over time

In [16]:
nominated_categories_per_year=oscar.groupby('year_ceremony')['category'].count().reset_index()
nominated_categories_per_year['change%']=round(nominated_categories_per_year['category'].pct_change()*100,2)
nominated_categories_per_year

year_ceremony  category  change%
0            1928        35      NaN
1            1929        38     8.57
2            1930        42    10.53
3            1931        44     4.76
4            1932        45     2.27
..            ...       ...      ...
91           2020       128     2.40
92           2021       120    -6.25
93           2022       124     3.33
94           2023       126     1.61
95           2024       124    -1.59

[96 rows x 3 columns]

In [17]:

fig = px.bar(nominated_categories_per_year, x='year_ceremony', y='category', title='Nominated Categories per year', height=1000)

# Add a line plot for the % change
fig.add_scatter(x=nominated_categories_per_year['year_ceremony'], y=nominated_categories_per_year['change%'], mode='lines', name='% Change')

# Update layout and axis labels
fig.update_layout(xaxis_title='Year', yaxis_title='Category Count')

# Show the plot
fig.show()

#c. Distinct Nominated Categories & Winning Proportion

In [18]:
category_counts=oscar['category'].value_counts().reset_index()
category_counts.columns = ['CATEGORY', 'Total Nominations']
category_counts

CATEGORY  Total Nominations
0                                    DIRECTING                469
1                                 FILM EDITING                450
2                 ACTRESS IN A SUPPORTING ROLE                440
3                   ACTOR IN A SUPPORTING ROLE                440
4                  DOCUMENTARY (Short Subject)                378
..                                         ...                ...
110        SPECIAL FOREIGN LANGUAGE FILM AWARD                  2
111                     GORDON E. SAWYER AWARD                  1
112  SPECIAL ACHIEVEMENT AWARD (Sound Editing)                  1
113  SPECIAL ACHIEVEMENT AWARD (Sound Effects)                  1
114                      AWARD OF COMMENDATION                  1

[115 rows x 2 columns]

In [19]:
winning_category_counts = oscar[oscar['winner'] == True].groupby('category').size().reset_index()
winning_category_counts.columns=['CATEGORY', 'Wins']
winning_category_counts

CATEGORY  Wins
0                                                ACTOR    49
1                              ACTOR IN A LEADING ROLE    48
2                           ACTOR IN A SUPPORTING ROLE    88
3                                              ACTRESS    49
4                            ACTRESS IN A LEADING ROLE    48
..                                                 ...   ...
110                     WRITING (Story and Screenplay)     7
111  WRITING (Story and Screenplay--based on factua...     4
112  WRITING (Story and Screenplay--based on materi...     1
113  WRITING (Story and Screenplay--written directl...    12
114                            WRITING (Title Writing)     1

[115 rows x 2 columns]

In [49]:
# Merge the two DataFrames on the 'Category' column
category_data = pd.merge(category_counts, winning_category_counts, on='CATEGORY', how='left')

# Calculate the proportion of nominations that resulted in winning an award for each category
category_data['Proportion of Wins'] = category_data['Wins'] / category_data['Total Nominations']*100

# Fill NaN values with 0 (if there were no wins in a category)
category_data.fillna(0, inplace=True)

# I want the top 20 categories
WIN_CATEGORIES=category_data.head(20)
WIN_CATEGORIES.sort_values(by='Proportion of Wins', ascending=False)

CATEGORY  Total Nominations  Wins  Proportion of Wins
6          DOCUMENTARY (Feature)                345    79           22.898551
18      SHORT FILM (Live Action)                226    50           22.123894
4    DOCUMENTARY (Short Subject)                378    80           21.164021
17                         ACTOR                232    49           21.120690
16         MUSIC (Original Song)                235    49           20.851064
15                       ACTRESS                236    49           20.762712
7                 CINEMATOGRAPHY                338    70           20.710059
0                      DIRECTING                469    95           20.255864
12                         SOUND                245    49           20.000000
14       ACTOR IN A LEADING ROLE                240    48           20.000000
13     ACTRESS IN A LEADING ROLE                240    48           20.000000
10                COSTUME DESIGN                295    59           20.000000
1                   FILM EDITING                450    90           20.000000
8          FOREIGN LANGUAGE FILM                315    63           20.000000
3     ACTOR IN A SUPPORTING ROLE                440    88           20.000000
2   ACTRESS IN A SUPPORTING ROLE                440    88           20.000000
9                  ART DIRECTION                307    59           19.218241
11        MUSIC (Original Score)                270    49           18.148148
19                  MUSIC (Song)                215    36           16.744186
5                   BEST PICTURE                371    62           16.711590

In [21]:
px.bar(WIN_CATEGORIES, x='CATEGORY', y=['Total Nominations', 'Wins'],
             title='Total Nominations vs Wins per Category',
             labels={'value': 'Count', 'variable': 'Metric', 'Category': 'Category'},
             color_discrete_map={'Total Nominations': 'black', 'Wins': 'gold'},
             barmode='group')

##d. Winning Proportion of movies

In [47]:
# Calculate the total number of films and the number of winning films for each year
films_per_year = oscar.groupby('year_film').size().reset_index(name='Total Films')
winning_films_per_year = oscar[oscar['winner'] == True].groupby('year_film').size().reset_index(name='Winning Films')

# Merge the two DataFrames on the 'year_film' column
films_data = pd.merge(films_per_year, winning_films_per_year, on='year_film', how='left')

# Calculate the proportion of winning films for each year
films_data['Winning Proportion'] = round(films_data['Winning Films'] / films_data['Total Films']*100,2)
films_data.sort_values(by='Winning Proportion', ascending=False)

year_film  Total Films  Winning Films  Winning Proportion
0        1927           35             15               42.86
4        1931           45             14               31.11
5        1932           62             19               30.65
20       1948          109             32               29.36
21       1949          112             32               28.57
..        ...          ...            ...                 ...
14       1942          186             30               16.13
16       1944          169             27               15.98
13       1941          183             28               15.30
12       1940          160             24               15.00
15       1943          182             26               14.29

[96 rows x 4 columns]

In [23]:
px.bar(films_data, x='year_film', y=['Total Films', 'Winning Films'], title='Total Nominations vs Wins per Film',
             labels={'value': 'Count', 'variable': 'Metric', 'year_film': 'Years'},
             color_discrete_map={'Total Films': 'black', 'Winning Films': 'gold'})

### Insights:

(a) A total of  **186 films** were released in **1942**, which is by far the highest number of releases.

(b) The year **1933** has witnessed highest increase in the no. of nominated categories (**+38%** approx.) on YoY basis; whereas year **1947** shown a sharp **decrease of 35%** of categories been nominated.

(c) **'Directing'** category has the **highest no. of nominations**, whereas **'short films' and 'documentaries' has the highest winning proportion** (22% each, approximately). **Directing** witnessed a winning proportion of **20%**.

(d) The year **1927** has the highest proportion of winning films (approximately 43%), followed by 1931, 1932.

#5. Winners and Nominees:

##a. Which films have won the most Oscars?
##b. Which individuals have won the most awards?
##c. Top 5 award winning movie analysis
##d. Top 5 award winning person/entity analysis

## (a) Which films have won the most Oscars?

In [24]:
winning_film=oscar[oscar['winner']==True]
# Group by film and count the number of wins for each film
wins_per_film = winning_film.groupby('film').size().reset_index(name='Wins')

# Sort the results in descending order based on the number of wins
wins_per_film_sorted = wins_per_film.sort_values(by='Wins', ascending=False)

# Display the films with the most wins
print("Films with the most Oscars:")
print(wins_per_film_sorted.head(15))

Films with the most Oscars:
                                               film  Wins
1215                                        Titanic    12
1274                                West Side Story    11
1055  The Lord of the Rings: The Return of the King    11
134                                         Ben-Hur    11
960                             The English Patient     9
380                                            Gigi     9
1037                               The Last Emperor     9
393                              Gone with the Wind     8
674                               On the Waterfront     8
68                                          Amadeus     8
365                           From Here to Eternity     8
199                                         Cabaret     8
804                             Slumdog Millionaire     8
369                                          Gandhi     8
638                                    My Fair Lady     8


## (b) Which individuals have won the most awards?

In [25]:
winning_person=oscar[oscar['winner']==True]
# Group by film and count the number of wins for each film
wins_per_person = winning_film.groupby('name').size().reset_index(name='Wins')

# Sort the results in descending order based on the number of wins
wins_per_person_sorted = wins_per_person.sort_values(by='Wins', ascending=False)

# Display the films with the most wins
print("Person with the most Oscars:")
print(wins_per_person_sorted.head(15))

Person with the most Oscars:
                                                   name  Wins
1994                              Walt Disney, Producer    22
1354                                Metro-Goldwyn-Mayer    12
973                                               Italy    10
771                                              France     9
2007                                       Warner Bros.     7
246                                       Alfred Newman     7
879                       Gordon Hollingshead, Producer     6
708                             Edward Selzer, Producer     5
1529                                          Paramount     5
1355  Metro-Goldwyn-Mayer Studio Sound Department, D...     5
1104                                          John Ford     4
1158                               Joseph L. Mankiewicz     4
1187                                  Katharine Hepburn     4
2044                                      William Wyler     4
585                                      

## (c) Top 5 Award Winning Movie Analysis

###1. Titanic

There are 2 movies named 'Titanic' (1953, 1997) in this dataset.

In [26]:
Titanic=oscar[(oscar['film']=='Titanic') & (oscar['year_film']==1953)].reset_index()
Titanic

index  year_film  year_ceremony  ceremony                         category  \
0   2780       1953           1954        26  ART DIRECTION (Black-and-White)   
1   2879       1953           1954        26   WRITING (Story and Screenplay)   

                                                name     film  winner  
0  Art Direction:  Lyle Wheeler, Maurice Ransford...  Titanic   False  
1     Charles Brackett, Walter Reisch, Richard Breen  Titanic    True

In [27]:
Titanic=oscar[(oscar['film']=='Titanic') & (oscar['winner']==True) & (oscar['year_film']==1997)].reset_index()
Titanic

index  year_film  year_ceremony  ceremony  \
0    7692       1997           1998        70   
1    7696       1997           1998        70   
2    7701       1997           1998        70   
3    7707       1997           1998        70   
4    7722       1997           1998        70   
5    7735       1997           1998        70   
6    7745       1997           1998        70   
7    7750       1997           1998        70   
8    7765       1997           1998        70   
9    7768       1997           1998        70   
10   7771       1997           1998        70   

                           category  \
0                     ART DIRECTION   
1                    CINEMATOGRAPHY   
2                    COSTUME DESIGN   
3                         DIRECTING   
4                      FILM EDITING   
5   MUSIC (Original Dramatic Score)   
6             MUSIC (Original Song)   
7                      BEST PICTURE   
8                             SOUND   
9             SOUND EFFECTS EDITING   
10                   VISUAL EFFECTS   

                                                 name     film  winner  
0   Art Direction:  Peter Lamont;  Set Decoration:...  Titanic    True  
1                                   Russell Carpenter  Titanic    True  
2                                    Deborah L. Scott  Titanic    True  
3                                       James Cameron  Titanic    True  
4       Conrad Buff, James Cameron, Richard A. Harris  Titanic    True  
5                                        James Horner  Titanic    True  
6       Music by James Horner; Lyric by Will Jennings  Titanic    True  
7             James Cameron and Jon Landau, Producers  Titanic    True  
8   Gary Rydstrom, Tom Johnson, Gary Summers, Mark...  Titanic    True  
9                     Tom Bellfort, Christopher Boyes  Titanic    True  
10  Robert Legato, Mark Lasoff, Thomas L. Fisher, ...  Titanic    True

In [28]:
#what are the unique categories for which titanic won an award?
Titanic['category'].value_counts()

ART DIRECTION                      1
CINEMATOGRAPHY                     1
COSTUME DESIGN                     1
DIRECTING                          1
FILM EDITING                       1
MUSIC (Original Dramatic Score)    1
MUSIC (Original Song)              1
BEST PICTURE                       1
SOUND                              1
SOUND EFFECTS EDITING              1
VISUAL EFFECTS                     1
Name: category, dtype: int64

In [29]:
#what are the years in which titanic won an award?
Titanic['year_ceremony'].value_counts()

1998    11
Name: year_ceremony, dtype: int64

###2. West Side Story

There're 2 movies sharing this same name. One was released in 1961, other in 2021.

**(a) 1961**

In [30]:
West_side_story=oscar[(oscar['film']=='West Side Story') & (oscar['year_film']==1961)].reset_index()
West_side_story

index  year_film  year_ceremony  ceremony  \
0    3719       1961           1962        34   
1    3733       1961           1962        34   
2    3743       1961           1962        34   
3    3753       1961           1962        34   
4    3763       1961           1962        34   
5    3768       1961           1962        34   
6    3780       1961           1962        34   
7    3795       1961           1962        34   
8    3805       1961           1962        34   
9    3820       1961           1962        34   
10   3827       1961           1962        34   

                                             category  \
0                          ACTOR IN A SUPPORTING ROLE   
1                        ACTRESS IN A SUPPORTING ROLE   
2                               ART DIRECTION (Color)   
3                              CINEMATOGRAPHY (Color)   
4                              COSTUME DESIGN (Color)   
5                                           DIRECTING   
6                                        FILM EDITING   
7                MUSIC (Scoring of a Musical Picture)   
8                                 BEST MOTION PICTURE   
9                                               SOUND   
10  WRITING (Screenplay--based on material from an...   

                                                 name             film  winner  
0                                     George Chakiris  West Side Story    True  
1                                         Rita Moreno  West Side Story    True  
2   Art Direction:  Boris Leven;  Set Decoration: ...  West Side Story    True  
3                                      Daniel L. Fapp  West Side Story    True  
4                                       Irene Sharaff  West Side Story    True  
5                         Robert Wise, Jerome Robbins  West Side Story    True  
6                                     Thomas Stanford  West Side Story    True  
7   Saul Chaplin, Johnny Green, Sid Ramin, Irwin K...  West Side Story    True  
8                               Robert Wise, Producer  West Side Story    True  
9   Todd-AO Sound Department, Fred Hynes, Sound Di...  West Side Story    True  
10                                      Ernest Lehman  West Side Story   False

**(b) 2021**

In [31]:
West_side_story=oscar[(oscar['film']=='West Side Story') & (oscar['year_film']==2021)].reset_index()
West_side_story

index  year_film  year_ceremony  ceremony                      category  \
0  10531       2021           2022        94  ACTRESS IN A SUPPORTING ROLE   
1  10544       2021           2022        94                CINEMATOGRAPHY   
2  10549       2021           2022        94                COSTUME DESIGN   
3  10554       2021           2022        94                     DIRECTING   
4  10599       2021           2022        94                  BEST PICTURE   
5  10604       2021           2022        94             PRODUCTION DESIGN   
6  10619       2021           2022        94                         SOUND   

                                                name             film  winner  
0                                      Ariana DeBose  West Side Story    True  
1                                    Janusz Kaminski  West Side Story   False  
2                                      Paul Tazewell  West Side Story   False  
3                                   Steven Spielberg  West Side Story   False  
4  Steven Spielberg and Kristie Macosko Krieger, ...  West Side Story   False  
5  Production Design: Adam Stockhausen; Set Decor...  West Side Story   False  
6  Tod A. Maitland, Gary Rydstrom, Brian Chumney,...  West Side Story   False

###3. The Lord of the Rings: The Return of the King

In [32]:
L_rings=oscar[(oscar['film']=='The Lord of the Rings: The Return of the King')].reset_index()
L_rings

index  year_film  year_ceremony  ceremony                      category  \
0    8366       2003           2004        76                 ART DIRECTION   
1    8376       2003           2004        76                COSTUME DESIGN   
2    8380       2003           2004        76                     DIRECTING   
3    8394       2003           2004        76                  FILM EDITING   
4    8402       2003           2004        76                        MAKEUP   
5    8409       2003           2004        76        MUSIC (Original Score)   
6    8411       2003           2004        76         MUSIC (Original Song)   
7    8415       2003           2004        76                  BEST PICTURE   
8    8434       2003           2004        76                  SOUND MIXING   
9    8438       2003           2004        76                VISUAL EFFECTS   
10   8443       2003           2004        76  WRITING (Adapted Screenplay)   

                                                 name  \
0   Art Direction: Grant Major; Set Decoration: Da...   
1                    Ngila Dickson and Richard Taylor   
2                                       Peter Jackson   
3                                       Jamie Selkirk   
4                       Richard Taylor and Peter King   
5                                        Howard Shore   
6   Music and Lyric by Fran Walsh and Howard Shore...   
7   Barrie M. Osborne, Peter Jackson and Fran Wals...   
8   Christopher Boyes, Michael Semanick, Michael H...   
9   Jim Rygiel, Joe Letteri, Randall William Cook ...   
10  Screenplay by Fran Walsh, Philippa Boyens & Pe...   

                                             film  winner  
0   The Lord of the Rings: The Return of the King    True  
1   The Lord of the Rings: The Return of the King    True  
2   The Lord of the Rings: The Return of the King    True  
3   The Lord of the Rings: The Return of the King    True  
4   The Lord of the Rings: The Return of the King    True  
5   The Lord of the Rings: The Return of the King    True  
6   The Lord of the Rings: The Return of the King    True  
7   The Lord of the Rings: The Return of the King    True  
8   The Lord of the Rings: The Return of the King    True  
9   The Lord of the Rings: The Return of the King    True  
10  The Lord of the Rings: The Return of the King    True

###4. Ben-Hur

In [33]:
Ben_hur=oscar[(oscar['film']=='Ben-Hur')].reset_index()
Ben_hur

index  year_film  year_ceremony  ceremony  \
0    3474       1959           1960        32   
1    3478       1959           1960        32   
2    3498       1959           1960        32   
3    3508       1959           1960        32   
4    3518       1959           1960        32   
5    3523       1959           1960        32   
6    3534       1959           1960        32   
7    3543       1959           1960        32   
8    3559       1959           1960        32   
9    3572       1959           1960        32   
10   3577       1959           1960        32   
11   3580       1959           1960        32   

                                             category  \
0                                               ACTOR   
1                          ACTOR IN A SUPPORTING ROLE   
2                               ART DIRECTION (Color)   
3                              CINEMATOGRAPHY (Color)   
4                              COSTUME DESIGN (Color)   
5                                           DIRECTING   
6                                        FILM EDITING   
7   MUSIC (Music Score of a Dramatic or Comedy Pic...   
8                                 BEST MOTION PICTURE   
9                                               SOUND   
10                                    SPECIAL EFFECTS   
11  WRITING (Screenplay--based on material from an...   

                                                 name     film  winner  
0                                     Charlton Heston  Ben-Hur    True  
1                                       Hugh Griffith  Ben-Hur    True  
2   Art Direction:  William A. Horning, Edward Car...  Ben-Hur    True  
3                                   Robert L. Surtees  Ben-Hur    True  
4                                 Elizabeth Haffenden  Ben-Hur    True  
5                                       William Wyler  Ben-Hur    True  
6                   Ralph E. Winters, John D. Dunning  Ben-Hur    True  
7                                        Miklos Rozsa  Ben-Hur    True  
8                             Sam Zimbalist, Producer  Ben-Hur    True  
9   Metro-Goldwyn-Mayer Studio Sound Department, F...  Ben-Hur    True  
10  Visual Effects by A. Arnold Gillespie, Robert ...  Ben-Hur    True  
11                                       Karl Tunberg  Ben-Hur   False

###5. The English Patient

In [34]:
Eng_patient=oscar[(oscar['film']=='The English Patient')].reset_index()
Eng_patient

index  year_film  year_ceremony  ceremony  \
0    7554       1996           1997        69   
1    7566       1996           1997        69   
2    7570       1996           1997        69   
3    7574       1996           1997        69   
4    7578       1996           1997        69   
5    7585       1996           1997        69   
6    7588       1996           1997        69   
7    7603       1996           1997        69   
8    7616       1996           1997        69   
9    7631       1996           1997        69   
10   7645       1996           1997        69   
11   7657       1996           1997        69   

                                             category  \
0                             ACTOR IN A LEADING ROLE   
1                           ACTRESS IN A LEADING ROLE   
2                        ACTRESS IN A SUPPORTING ROLE   
3                                       ART DIRECTION   
4                                      CINEMATOGRAPHY   
5                                      COSTUME DESIGN   
6                                           DIRECTING   
7                                        FILM EDITING   
8                     MUSIC (Original Dramatic Score)   
9                                        BEST PICTURE   
10                                              SOUND   
11  WRITING (Screenplay Based on Material Previous...   

                                                 name                 film  \
0                                       Ralph Fiennes  The English Patient   
1                                Kristin Scott Thomas  The English Patient   
2                                    Juliette Binoche  The English Patient   
3   Art Direction:  Stuart Craig;  Set Decoration:...  The English Patient   
4                                          John Seale  The English Patient   
5                                            Ann Roth  The English Patient   
6                                   Anthony Minghella  The English Patient   
7                                        Walter Murch  The English Patient   
8                                       Gabriel Yared  The English Patient   
9                               Saul Zaentz, Producer  The English Patient   
10  Walter Murch, Mark Berger, David Parker, Chris...  The English Patient   
11                                  Anthony Minghella  The English Patient   

    winner  
0    False  
1    False  
2     True  
3     True  
4     True  
5     True  
6     True  
7     True  
8     True  
9     True  
10    True  
11   False

##(d). Topmost award winning person/entity analysis

In [35]:
WD=oscar[(oscar['name']=='Walt Disney, Producer') & (oscar['winner']==True)].reset_index()
WD

index  year_film  year_ceremony  ceremony                     category  \
0     182       1931           1932         5      SHORT SUBJECT (Cartoon)   
1     249       1932           1933         6      SHORT SUBJECT (Cartoon)   
2     308       1934           1935         7      SHORT SUBJECT (Cartoon)   
3     389       1935           1936         8      SHORT SUBJECT (Cartoon)   
4     489       1936           1937         9      SHORT SUBJECT (Cartoon)   
5     610       1937           1938        10      SHORT SUBJECT (Cartoon)   
6     740       1938           1939        11      SHORT SUBJECT (Cartoon)   
7     885       1939           1940        12      SHORT SUBJECT (Cartoon)   
8    1211       1941           1942        14      SHORT SUBJECT (Cartoon)   
9    1404       1942           1943        15      SHORT SUBJECT (Cartoon)   
10   2261       1948           1949        21     SHORT SUBJECT (Two-reel)   
11   2481       1950           1951        23     SHORT SUBJECT (Two-reel)   
12   2602       1951           1952        24     SHORT SUBJECT (Two-reel)   
13   2727       1952           1953        25     SHORT SUBJECT (Two-reel)   
14   2812       1953           1954        26        DOCUMENTARY (Feature)   
15   2814       1953           1954        26  DOCUMENTARY (Short Subject)   
16   2848       1953           1954        26      SHORT SUBJECT (Cartoon)   
17   2854       1953           1954        26     SHORT SUBJECT (Two-reel)   
18   2941       1954           1955        27        DOCUMENTARY (Feature)   
19   3071       1955           1956        28  DOCUMENTARY (Short Subject)   
20   3449       1958           1959        31  SHORT SUBJECT (Live Action)   
21   4643       1968           1969        41      SHORT SUBJECT (Cartoon)   

                     name                                  film  winner  
0   Walt Disney, Producer                     Flowers and Trees    True  
1   Walt Disney, Producer                 The Three Little Pigs    True  
2   Walt Disney, Producer             The Tortoise and the Hare    True  
3   Walt Disney, Producer                  Three Orphan Kittens    True  
4   Walt Disney, Producer                    The Country Cousin    True  
5   Walt Disney, Producer                          The Old Mill    True  
6   Walt Disney, Producer                    Ferdinand the Bull    True  
7   Walt Disney, Producer                     The Ugly Duckling    True  
8   Walt Disney, Producer                            Lend a Paw    True  
9   Walt Disney, Producer                    Der Fuehrer's Face    True  
10  Walt Disney, Producer                           Seal Island    True  
11  Walt Disney, Producer                      In Beaver Valley    True  
12  Walt Disney, Producer                    Nature's Half Acre    True  
13  Walt Disney, Producer                           Water Birds    True  
14  Walt Disney, Producer                     The Living Desert    True  
15  Walt Disney, Producer                    The Alaskan Eskimo    True  
16  Walt Disney, Producer         Toot, Whistle, Plunk and Boom    True  
17  Walt Disney, Producer                          Bear Country    True  
18  Walt Disney, Producer                 The Vanishing Prairie    True  
19  Walt Disney, Producer                Men against the Arctic    True  
20  Walt Disney, Producer                          Grand Canyon    True  
21  Walt Disney, Producer  Winnie the Pooh and the Blustery Day    True

In [36]:
WD['category'].value_counts()

SHORT SUBJECT (Cartoon)        12
SHORT SUBJECT (Two-reel)        5
DOCUMENTARY (Feature)           2
DOCUMENTARY (Short Subject)     2
SHORT SUBJECT (Live Action)     1
Name: category, dtype: int64

In [37]:
MGM=oscar[(oscar['name']=='Metro-Goldwyn-Mayer') & (oscar['winner']==True)].reset_index()
MGM

index  year_film  year_ceremony  ceremony                    category  \
0      64       1928           1929         2         OUTSTANDING PICTURE   
1     178       1931           1932         5      OUTSTANDING PRODUCTION   
2     384       1935           1936         8      OUTSTANDING PRODUCTION   
3     481       1936           1937         9      OUTSTANDING PRODUCTION   
4     500       1936           1937         9    SHORT SUBJECT (Two-reel)   
5     619       1937           1938        10    SHORT SUBJECT (Two-reel)   
6     745       1938           1939        11    SHORT SUBJECT (One-reel)   
7    1035       1940           1941        13     SHORT SUBJECT (Cartoon)   
8    1223       1941           1942        14    SHORT SUBJECT (One-reel)   
9    1228       1941           1942        14    SHORT SUBJECT (Two-reel)   
10   1395       1942           1943        15  OUTSTANDING MOTION PICTURE   
11   2733       1952           1953        25             SPECIAL EFFECTS   

                   name                       film  winner  
0   Metro-Goldwyn-Mayer        The Broadway Melody    True  
1   Metro-Goldwyn-Mayer                Grand Hotel    True  
2   Metro-Goldwyn-Mayer       Mutiny on the Bounty    True  
3   Metro-Goldwyn-Mayer         The Great Ziegfeld    True  
4   Metro-Goldwyn-Mayer            The Public Pays    True  
5   Metro-Goldwyn-Mayer              Torture Money    True  
6   Metro-Goldwyn-Mayer    That Mothers Might Live    True  
7   Metro-Goldwyn-Mayer              The Milky Way    True  
8   Metro-Goldwyn-Mayer        Of Pups and Puzzles    True  
9   Metro-Goldwyn-Mayer  Main Street on the March!    True  
10  Metro-Goldwyn-Mayer               Mrs. Miniver    True  
11  Metro-Goldwyn-Mayer         Plymouth Adventure    True

In [38]:
MGM['category'].value_counts()

OUTSTANDING PRODUCTION        3
SHORT SUBJECT (Two-reel)      3
SHORT SUBJECT (One-reel)      2
OUTSTANDING PICTURE           1
SHORT SUBJECT (Cartoon)       1
OUTSTANDING MOTION PICTURE    1
SPECIAL EFFECTS               1
Name: category, dtype: int64

### Insights:

### **MOST AWARD-WINNING MOVIES:**
## (a) There are 2 movies named 'Titanic' (1953, 1997) in this dataset.

The **1997's Titanic** was **nominated** for oscars **14 times**, out of which it **won 11 times** (Each in 11 distinct categories) in the year 1998.

The **1953's Titanic** was **nominated twice** in 1954 and **won an award for "WRITING (Story and Screenplay)**". This award was shared amongst Charles Brackett, Walter Reisch, Richard Breen.

## (b) There're 2 movies named 'West Side Story'. One was released in 1961, other in 2021.

The **1961's West Side Story** was **nominated** for oscars **11 times**, out of which it **won 10 times** (Each in 10 distinct categories) in the year 1962.

The **2021's West Side Story** was **nominated 7 times** in 2022 and **won an award for "ACTRESS IN A SUPPORTING ROLE" by Ariana DeBose.**

## (c) 2003's "**The Lord of the Rings: The Return of the King**" was nominated 11 times for oscars and won all the 11 times (each in distinct 11 categories)

## (d) 1959's **Ben-Hur** was nominated for 12 times and won 11 awards for each distinct category.

## (e) 1996's **The English Patient** was nominated 12 times, wherein it won 9 awards.

### **MOST AWARD-WINNING INDIVIDUALS/ENTITIES:**

##(a) **Walt Disney Productions** were nominated 59 times, out of which it won 22 times. Most of which (almost 55%) were won for SHORT SUBJECT (Cartoon) category.

##(b) **MGM** was nominated for 64 times, out of which it won 12 times. Mostly for Production & short subject content

##6. Recent Trends (2010s- till present)

We want to shed light onto the trends of oscar nominees and winners over the period: 2010 to 2023.

In [39]:
Recent=oscar[(oscar['year_film']>2009) & (oscar['winner']==True)].reset_index()
Recent

index  year_film  year_ceremony  ceremony  \
0     9145       2010           2011        83   
1     9147       2010           2011        83   
2     9155       2010           2011        83   
3     9159       2010           2011        83   
4     9164       2010           2011        83   
..     ...        ...            ...       ...   
383  10880       2023           2024        96   
384  10885       2023           2024        96   
385  10886       2023           2024        96   
386  10887       2023           2024        96   
387  10888       2023           2024        96   

                             category  \
0             ACTOR IN A LEADING ROLE   
1          ACTOR IN A SUPPORTING ROLE   
2           ACTRESS IN A LEADING ROLE   
3        ACTRESS IN A SUPPORTING ROLE   
4               ANIMATED FEATURE FILM   
..                                ...   
383     WRITING (Original Screenplay)   
384  JEAN HERSHOLT HUMANITARIAN AWARD   
385                    HONORARY AWARD   
386                    HONORARY AWARD   
387                    HONORARY AWARD   

                                                  name               film  \
0                                          Colin Firth  The King's Speech   
1                                       Christian Bale        The Fighter   
2                                      Natalie Portman         Black Swan   
3                                          Melissa Leo        The Fighter   
4                                          Lee Unkrich        Toy Story 3   
..                                                 ...                ...   
383       Screenplay - Justine Triet and Arthur Harari  Anatomy of a Fall   
384                                                NaN                NaN   
385  To Angela Bassett, who has inspired audiences ...                NaN   
386  To Mel Brooks, for his comedic brilliance, pro...                NaN   
387  To Carol Littleton, whose commitment to her cr...                NaN   

     winner  
0      True  
1      True  
2      True  
3      True  
4      True  
..      ...  
383    True  
384    True  
385    True  
386    True  
387    True  

[388 rows x 8 columns]

In [40]:
#which category won the highest number of awards
Recent_wins=Recent[Recent['winner']==True]
Recent_cats=Recent_wins['category'].value_counts()
Recent_cats

HONORARY AWARD                       39
ACTOR IN A LEADING ROLE              14
FILM EDITING                         14
WRITING (Original Screenplay)        14
WRITING (Adapted Screenplay)         14
VISUAL EFFECTS                       14
ACTOR IN A SUPPORTING ROLE           14
SHORT FILM (Animated)                14
BEST PICTURE                         14
MUSIC (Original Song)                14
MUSIC (Original Score)               14
SHORT FILM (Live Action)             14
ACTRESS IN A LEADING ROLE            14
DIRECTING                            14
COSTUME DESIGN                       14
CINEMATOGRAPHY                       14
ANIMATED FEATURE FILM                14
ACTRESS IN A SUPPORTING ROLE         14
DOCUMENTARY (Short Subject)          12
DOCUMENTARY (Feature)                12
PRODUCTION DESIGN                    12
MAKEUP AND HAIRSTYLING               12
JEAN HERSHOLT HUMANITARIAN AWARD     11
SOUND EDITING                        11
SOUND MIXING                         10


In [41]:
#Most award-winning film
Recent['film'].value_counts()

Oppenheimer                          7
Gravity                              7
Everything Everywhere All at Once    7
Mad Max: Fury Road                   6
La La Land                           6
                                    ..
Inocente                             1
Searching for Sugar Man              1
Phantom Thread                       1
Anna Karenina                        1
Anatomy of a Fall                    1
Name: film, Length: 205, dtype: int64

In [42]:
#Most award-winning person
Recent['name'].value_counts()

Alfonso Cuarón                                                                                           3
Emmanuel Lubezki                                                                                         3
Jenny Beavan                                                                                             2
Emma Stone                                                                                               2
Alejandro G. Iñárritu                                                                                    2
                                                                                                        ..
Music and Lyric by John Stephens and Lonnie Lynn                                                         1
Frances Hannon and Mark Coulier                                                                          1
Poland                                                                                                   1
Tom Cross                            

In [43]:
#Across which all categories did Oppenheimer win an award?
Opp=Recent[Recent['film']=='Oppenheimer']
Opp

index  year_film  year_ceremony  ceremony                    category  \
361  10768       2023           2024        96     ACTOR IN A LEADING ROLE   
362  10772       2023           2024        96  ACTOR IN A SUPPORTING ROLE   
366  10793       2023           2024        96              CINEMATOGRAPHY   
368  10802       2023           2024        96                   DIRECTING   
371  10818       2023           2024        96                FILM EDITING   
374  10833       2023           2024        96      MUSIC (Original Score)   
376  10846       2023           2024        96                BEST PICTURE   

                                                  name         film  winner  
361                                     Cillian Murphy  Oppenheimer    True  
362                                  Robert Downey Jr.  Oppenheimer    True  
366                                  Hoyte van Hoytema  Oppenheimer    True  
368                                  Christopher Nolan  Oppenheimer    True  
371                                      Jennifer Lame  Oppenheimer    True  
374                                   Ludwig Göransson  Oppenheimer    True  
376  Emma Thomas, Charles Roven and Christopher Nol...  Oppenheimer    True

In [44]:
#Most award winning director?
Director=Recent[Recent['category']=='DIRECTING']
Dir_wins=Director['name'].value_counts()
Dir_wins

Alfonso Cuarón                      2
Alejandro G. Iñárritu               2
Tom Hooper                          1
Michel Hazanavicius                 1
Ang Lee                             1
Damien Chazelle                     1
Guillermo del Toro                  1
Bong Joon Ho                        1
Chloé Zhao                          1
Jane Campion                        1
Daniel Kwan and Daniel Scheinert    1
Christopher Nolan                   1
Name: name, dtype: int64

### Insights:

1. **'Honorary Award'** is the most award winning category (39 awards), in recent times.
2. **'Oppenheimer', 'Gravity', 'Everything Everywhere All at Once'** is most award winning movie; bagging **7** **awards**, each.

Oppenheimer (2023) won awards across 7 distinct categories:

**ACTOR IN A LEADING ROLE (Cillian Murphy), ACTOR IN A SUPPORTING ROLE (Robert Downey Jr), CINEMATOGRAPHY (Hoyte van Hoytema), DIRECTING (Christopher Nolan), FILM EDITING (Jennifer Lame), MUSIC Original Score (Ludwig Göransson), BEST PICTURE (the crew).**


3. **Alfonso Cuarón** is the most award-winning individual, of recent times; winning **3 awards** since 2010.




# **Winners of 2023!!**

In [50]:
#2023 films winning an oscar
entries_2023=Recent[Recent['year_film']==2023]
entries_2023

index  year_film  year_ceremony  ceremony  \
361  10768       2023           2024        96   
362  10772       2023           2024        96   
363  10779       2023           2024        96   
364  10784       2023           2024        96   
365  10785       2023           2024        96   
366  10793       2023           2024        96   
367  10799       2023           2024        96   
368  10802       2023           2024        96   
369  10809       2023           2024        96   
370  10813       2023           2024        96   
371  10818       2023           2024        96   
372  10824       2023           2024        96   
373  10828       2023           2024        96   
374  10833       2023           2024        96   
375  10839       2023           2024        96   
376  10846       2023           2024        96   
377  10854       2023           2024        96   
378  10859       2023           2024        96   
379  10864       2023           2024        96   
380  10869       2023           2024        96   
381  10871       2023           2024        96   
382  10875       2023           2024        96   
383  10880       2023           2024        96   
384  10885       2023           2024        96   
385  10886       2023           2024        96   
386  10887       2023           2024        96   
387  10888       2023           2024        96   

                             category  \
361           ACTOR IN A LEADING ROLE   
362        ACTOR IN A SUPPORTING ROLE   
363         ACTRESS IN A LEADING ROLE   
364      ACTRESS IN A SUPPORTING ROLE   
365             ANIMATED FEATURE FILM   
366                    CINEMATOGRAPHY   
367                    COSTUME DESIGN   
368                         DIRECTING   
369          DOCUMENTARY FEATURE FILM   
370            DOCUMENTARY SHORT FILM   
371                      FILM EDITING   
372        INTERNATIONAL FEATURE FILM   
373            MAKEUP AND HAIRSTYLING   
374            MUSIC (Original Score)   
375             MUSIC (Original Song)   
376                      BEST PICTURE   
377                 PRODUCTION DESIGN   
378             SHORT FILM (Animated)   
379          SHORT FILM (Live Action)   
380                             SOUND   
381                    VISUAL EFFECTS   
382      WRITING (Adapted Screenplay)   
383     WRITING (Original Screenplay)   
384  JEAN HERSHOLT HUMANITARIAN AWARD   
385                    HONORARY AWARD   
386                    HONORARY AWARD   
387                    HONORARY AWARD   

                                                  name  \
361                                     Cillian Murphy   
362                                  Robert Downey Jr.   
363                                         Emma Stone   
364                               Da'Vine Joy Randolph   
365                   Hayao Miyazaki and Toshio Suzuki   
366                                  Hoyte van Hoytema   
367                                   Holly Waddington   
368                                  Christopher Nolan   
369  Mstyslav Chernov, Michelle Mizner and Raney Ar...   
370                      Ben Proudfoot and Kris Bowers   
371                                      Jennifer Lame   
372                                     United Kingdom   
373         Nadia Stacey, Mark Coulier and Josh Weston   
374                                   Ludwig Göransson   
375  Music and Lyric by Billie Eilish and Finneas O...   
376  Emma Thomas, Charles Roven and Christopher Nol...   
377  Production Design: James Price and Shona Heath...   
378                       Dave Mullins and Brad Booker   
379                      Wes Anderson and Steven Rales   
380                      Tarn Willers and Johnnie Burn   
381  Takashi Yamazaki, Kiyoko Shibuya, Masaki Takah...   
382           Written for the screen by Cord Jefferson   
383       Screenplay - Justine Triet and Arthur Harari   
384                                                NaN   
385  

In [51]:
#no. of oscars per movie
entries_2023['film'].value_counts()

Oppenheimer                                          7
Poor Things                                          4
The Zone of Interest                                 2
The Holdovers                                        1
The Boy and the Heron                                1
20 Days in Mariupol                                  1
The Last Repair Shop                                 1
Barbie                                               1
WAR IS OVER! Inspired by the Music of John & Yoko    1
The Wonderful Story of Henry Sugar                   1
Godzilla Minus One                                   1
American Fiction                                     1
Anatomy of a Fall                                    1
Name: film, dtype: int64

In [52]:
#categories with highest no. of oscars
entries_2023['category'].value_counts().sort_values(ascending=False)

HONORARY AWARD                      3
ACTOR IN A SUPPORTING ROLE          1
ACTRESS IN A LEADING ROLE           1
ACTRESS IN A SUPPORTING ROLE        1
ANIMATED FEATURE FILM               1
CINEMATOGRAPHY                      1
COSTUME DESIGN                      1
DIRECTING                           1
DOCUMENTARY FEATURE FILM            1
DOCUMENTARY SHORT FILM              1
FILM EDITING                        1
INTERNATIONAL FEATURE FILM          1
ACTOR IN A LEADING ROLE             1
MUSIC (Original Score)              1
MUSIC (Original Song)               1
BEST PICTURE                        1
PRODUCTION DESIGN                   1
SHORT FILM (Animated)               1
SHORT FILM (Live Action)            1
SOUND                               1
VISUAL EFFECTS                      1
WRITING (Adapted Screenplay)        1
WRITING (Original Screenplay)       1
JEAN HERSHOLT HUMANITARIAN AWARD    1
MAKEUP AND HAIRSTYLING              1
Name: category, dtype: int64

In [54]:
#individuals/entities with highest number of awards
entries_2023['name'].value_counts().sort_values(ascending=True)

Cillian Murphy                                                                                                 1
Da'Vine Joy Randolph                                                                                           1
Hayao Miyazaki and Toshio Suzuki                                                                               1
Hoyte van Hoytema                                                                                              1
Holly Waddington                                                                                               1
Christopher Nolan                                                                                              1
Mstyslav Chernov, Michelle Mizner and Raney Aronson-Rath                                                       1
Ben Proudfoot and Kris Bowers                                                                                  1
Jennifer Lame                                                                                   

### Insights:
1. Oppenheimer (7 awards), Poor things (3 awards), The zone of interest (2 awards) are most award winning movies.

2. Honorary award is the most award-winning category.



## Oscar-winning movie per Decade
Finally, What are the number of oscar winning movie in each decade?

In [45]:
import pandas as pd

# Assuming 'oscar' is your DataFrame containing the Oscars data

# 1. Filter the dataset to keep only the records where 'winner' is True
oscar_winners = oscar[oscar['winner'] == True]

# 2. Create groups based on the conditions for 'year_film'
conditions = [
    (oscar_winners['year_film'] >= 1920) & (oscar_winners['year_film'] <= 1929),
    (oscar_winners['year_film'] >= 1930) & (oscar_winners['year_film'] <= 1939),
    (oscar_winners['year_film'] >= 1940) & (oscar_winners['year_film'] <= 1949),
    (oscar_winners['year_film'] >= 1950) & (oscar_winners['year_film'] <= 1959),
    (oscar_winners['year_film'] >= 1960) & (oscar_winners['year_film'] <= 1969),
    (oscar_winners['year_film'] >= 1970) & (oscar_winners['year_film'] <= 1979),
    (oscar_winners['year_film'] >= 1980) & (oscar_winners['year_film'] <= 1989),
    (oscar_winners['year_film'] >= 1990) & (oscar_winners['year_film'] <= 1999),
    (oscar_winners['year_film'] >= 2000) & (oscar_winners['year_film'] <= 2009),
    (oscar_winners['year_film'] >= 2010) & (oscar_winners['year_film'] <= 2019),
    (oscar_winners['year_film'] >= 2020)
]

# Define the corresponding groups
groups = ['1920s', '1930s', '1940s', '1950s', '1960s', '1970s', '1980s', '1990s', '2000s', '2010s', '2020s']

# Create a new column 'decade_group' based on the conditions and groups
oscar_winners['decade_group'] = pd.cut(oscar_winners['year_film'], bins=[1919, 1929, 1939, 1949, 1959, 1969, 1979, 1989, 1999, 2009, 2019, 2029], labels=groups)

# 3. Count the unique number of films in each group
group_counts = oscar_winners.groupby('decade_group')['film'].nunique().reset_index()
group_counts.columns = ['Decade Group', 'Number of Films']

# Display the unique number of films in each group
group_counts

<ipython-input-45-7fd5f771dc33>:27: SettingWithCopyWarning:


A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy



Decade Group  Number of Films
0         1920s               20
1         1930s              101
2         1940s              170
3         1950s              149
4         1960s              143
5         1970s              134
6         1980s              135
7         1990s              138
8         2000s              147
9         2010s              148
10        2020s               57

In [46]:
px.bar(group_counts, x='Decade Group', y='Number of Films', color='Number of Films', title="Oscar-Winning Movies per Decade")

## Finally, 1940s (spanning 1940-1949) is the most award winning decade, with 170 sistinct films winning awards across different categories.

# **SUMMARY**

In this comprehensive exploratory data analysis project focusing on the Oscars, **we aimed to discern the trends and patterns of winning categories in Oscar-nominated films over the past 96 years.** Our investigation yielded several significant insights:

* Firstly, we found that a total of 5,042 movies were nominated across 115 distinct categories, involving 7,040 individuals. Among these, 1,329 films emerged victorious, reflecting a winning proportion of 26.0%. Interestingly, all nominated categories have secured awards at various points in time, and approximately 30% of the individuals nominated have clinched an award.

* Moreover, our analysis revealed that about 22.64% of nominations have culminated in receiving the prestigious accolade, underlining the selective and competitive nature of the Oscars. Notably, the year 1942 witnessed the highest number of film releases, while 1933 experienced a substantial increase in nominated categories, whereas 1947 saw a sharp decline.

* Furthermore, we identified 'Directing' as the category with the highest number of nominations, whereas 'short films' and 'documentaries' boasted the highest winning proportions (approximately 22% each). The years 1927, 1931, and 1932 stood out for their exceptionally high proportions of winning films.

**Delving into the most award-winning movies:**

> we highlighted notable films such as: **'Titanic,' 'West Side Story,' 'The Lord of the Rings: The Return of the King,' and 'Ben-Hur,'** among others. Additionally, we recognized Walt Disney Productions and MGM as the most award-winning entities, with their notable contributions to cinema.



> Moreover, **'Honorary Award' emerged as the most award-winning category**, while 'Oppenheimer,' 'Gravity,' and 'Everything Everywhere All at Once' claimed distinction as the most award-winning movies. Alfonso Cuarón emerged as the most award-winning individual of recent times (2010 - 2023).

* Finally, we concluded that the 1940s emerged as the most award-winning decade, with 170 distinct films clinching awards across various categories, encapsulating the rich history and enduring legacy of cinematic excellence celebrated by the Oscars over the years.